src/test/groovy/FilebeatStepTests.groovy

```diff
@@ -22,11 +22,14 @@ import static org.junit.Assert.assertFalse
 
 class FilebeatStepTests extends ApmBasePipelineTest {
   String scriptName = 'vars/filebeat.groovy'
+  // test resources file uses this value as filename
+  String nodeName = 'worker-0676d01d9601f8191'
```

Comment:
It appears that you are always using this to compose the same string, which is `filebeat_container_worker-0676d01d9601f8191.json`. Would it be better to just define it once and re-use it?

Reply:
Done!